### PR TITLE
Fixes

### PR DESCRIPTION
**Pull request by beschoenen**

### CODE REVIEW

**Comment by SickRage2** [comment]
Thanks for the pull request! Before a real human comes by, please make sure your PR has all of the below criteria checked

- [ ] Give a description on what the PR is for.
- [x] Make sure your PR is based on the DEVELOP branch
- [x] Don't send big changes all at once. Split up big PRs into multiple smaller PRs that are easier to manage and review

Please make sure you also read [contribution guide](https://github.com/SickRage/SickRage/blob/master/contributing.md) and followed all of the steps.

Thanks!

**Comment by beschoenen** [comment]
@miigotu I'm not sure what capitalization to use across SickRage, every settings page was different, so I made it all lowercase.

File: gui/slick/views/config_general.mako [hunk 1]
```diff
@@ -542,16 +543,18 @@
                             </div>
                             <div class="col-lg-9 col-md-8 col-sm-7 col-xs-12 component-desc">
                                 <label for="local" class="space-right">
-                                    <input type="radio" name="timezone_display" id="local" value="local" ${('', 'checked="checked"')[sickbeard.TIMEZONE_DISPLAY == "local"]} />${_('Local')}
+                                    <input type="radio" name="timezone_display" id="local" value="local" ${('', 'checked="checked"')[sickbeard.TIMEZONE_DISPLAY == "local"]} >
+                                    <label>${_('Local')}</label>
```
Comment (miigotu) [inline gui/slick/views/config_general.mako]: I think this is wrong, shoulndt the input be moved out of the label and not have a label.label ?
same for network below

Comment (beschoenen) [reply gui/slick/views/config_general.mako]: You're absolutely right

**Comment by miigotu** [comment]
About the capitalization, I prefer capitalizing the labels but this is ok for now

**Comment by miigotu** [comment]
Approved

[Approved with PullApprove](https://pullapprove.com/SickRage/SickRage/pull-request/2946/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=SickRage/SickRage)

**Review by miigotu — APPROVED** [review]

**Pull request merged**